src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -162,21 +164,19 @@ export default {
 
 	setup() {
 		const settingsStore = useSettingsStore()
+		const settingsToken = ref('')
```

Comment:
The new name adds a feeling that it is something new to me, like we have a conversation token, but also some other kind of tokens.
```suggestion
		const token = ref('')
```

It also allows using shortcuts in Vue 3.

Reply:
What do you think?

Reply:
👀
```js
token.value = payload.token
token.value = event.token
```